examples/index.json

```diff
@@ -295,7 +295,7 @@
                 "isLearnByExample": true
             },
             {
-                "name": "Controlling openness",
+                "name": "Control openness",
                 "url": "controlling-openness",
```

Comment:
Shouldn't these also change?

Reply:
Yes. `Controlling openness` has been changed to `Control openness`. Do you mean we should change the URL as well?

Reply:
Yes.

Reply:
Updated the URLs.